lib/actions/reassemble.js

```diff
@@ -1,12 +1,14 @@
 const { messages } = require('elasticio-node');
 
 const groupsSeen = {};
+const incomingData = {};
```

Comment:
It is possible for there to be two or more different groups being re-assembled at the same time. Therefore, when you store message data, you need to have it keyed on the groupId and then keyed on the messageId.

Reply:
possible done

Reply:
I think the trick is:
* Remove the `incomingData` as a global variable here
* Add `incomingData` as a new object property to [`groupsSeen[groupId]`](https://github.com/elasticio/splitter-component/blob/7eadec570744872b4ad3f2f6ea01e719498d3f80/lib/actions/reassemble.js#L17-L20)
* When emitting, the `messageData` property should be set to `groupsSeen[groupId].incomingData`

Reply:
Thanks, it's pretty clear.
Fixed